docs/schema-v2.md

```diff
@@ -5,12 +5,413 @@ This specification defines a format for a set of rules for the purpose of custom
+        "type": /* blend type (string, optional) */,
+        
+        // OR
+        
+        "sFactor": /* sFactor number (int, optional) */,
+        "dFactor": /* dFactor number (int, optional) */,
+        "equation": /* equation number (int, optional) */
+    },
+    "textures": // textures object (square-textured type only)
+    {
+        "north": /* texture to use for north direction (string, optional) */,
+        "south": /* texture to use for south direction (string, optional) */,
+        "east": /* texture to use for east direction (string, optional) */,
+        "west": /* texture to use for west direction (string, optional) */,
+        "top": /* texture to use for top direction (string, optional) */,
+        "bottom": /* texture to use for bottom direction (string, optional) */
+    },
+    "texture": /* path to single-sprite texture (string, single-sprite-square-textured type only) */,
+    "fps": /* frames per second for animation (float, animated-square-textured, single-sprite-animated-square-textured types only) */,
+    "animationTextures": // textures to use for animation (animated-square-textured type only)
+    [
+        { // textures for first frame
+            "north": /* texture to use for north direction (string, optional) */,
+            "south": /* texture to use for south direction (string, optional) */,
+            "east": /* texture to use for east direction (string, optional) */,
+            "west": /* texture to use for west direction (string, optional) */,
+            "top": /* texture to use for top direction (string, optional) */,
+            "bottom": /* texture to use for bottom direction (string, optional) */
+        }
+        // ...
+    ],
+    "animationTextures": // textures to use for animation (single-sprite-animated-square-textured type only)
+    [
+        /* single sprite texture for first frame (string) */
+        // ...
+    ]
+}
+```
+Note that this isn't actually a valid json file as some fields are blank and there are comments. Instead it serves to illustrate the general structure of a skybox file and gives basic descriptions of each item.  
+
+
+## Schema Versions
 | Version |    Date    |
 |:-------:|:----------:|
 |    2    | 10/14/2020 |
 
+
+## Skyboxes
+### Types
+There currently exist 5 types of skyboxes:
+|                   |  Monocolor  |            Textured             |             Animated Textured            |
+|:------------------|:-----------:|:-------------------------------:|:----------------------------------------:|
+|     **Normal**    | `monocolor` |        `square-textured`        |        `animated-square-textured`        |
+| **Single Sprite** |      -      | `single-sprite-square-textured` | `single-sprite-animated-square-textured` |
+
+Normal textured skyboxes require 6 image files (1 for each direction), and is recommended. Single sprite textured skyboxes only require 1 image file and follows the optifine specification, but is sometimes buggy.
```

Comment:
I believe it should be `follow` with no `s`.  Also `are` instead of `is` because the subject is plural.

Reply:
I changed it to "Single sprite textured skyboxes only require 1 image file **which** follows the optifine specification", as the skybox file itself still uses the fabricskyboxes schema. I think it's a little more clear

Reply:
I agree, however the grammatical problem there is simply the use of `follows` instead of follow`.

Reply:
NVM, all good.